docs/manual/common/concepts/PolyglotSystems.md

```diff
@@ -1,31 +1,15 @@
-
-However, in a microservice based system, not every service is expected to be a Lagom service.  After all, a big advantage of using microservices in the first place is to allow the construction of polyglot systems, selecting the right language and the right technology on a service by service basis.
-
-So while Lagom does provide a great experience when all the services are Lagom services, we do not want to imply that all services should be Lagom services. (In fact, if a large system was implemented using only Lagom services, then perhaps the technical merits of that system should be questioned...?)
-
-Consequently, Lagom is designed to work well in a polyglot environment. Central to ensuring this is the idiomatic usage of standard protocols.
+ Lagom does not expect that every service in your system will be a Lagom microservice.  After all, a big advantage of using microservices in the first place is to pragmatically select the best language and technology for each service. Lagom's idiomatic usage of standard comunication protocols and general compatibility with design driven API approaches allow it to work well in such a polyglot system.
 
 ## Lagom communication protocols
 
-Lagom service calls map down to ordinary, standard HTTP and WebSocket requests: plain HTTP for synchronous communication, WebSockets for streaming and asynchronous messaging.
+Lagom service calls map down to standard HTTP for synchronous communication and WebSockets for streaming and asynchronous messaging. Any language or framework that supports these protocols can easily consume a Lagom service.  Conversely, a Lagom service can easily talk to any service that exposes a REST API.
 
-A service's API specifies how that service uses HTTP. REST service calls are identified by an HTTP method and URI. Request and response headers can be customized.
-
-Lagom messages are serialised, by default, to ordinary JSON, using idiomatic mapping libraries that make it transparent how the JSON will be represented on the wire.
-
-The use of ordinary HTTP, WebSockets, and JSON means that any language or framework that supports these protocols can easily consume a Lagom service.  Conversely, a Lagom service can easily talk to any service that offers a REST API.
+A microservice's API specifies how that service uses HTTP. REST service calls are identified by an HTTP method and URI. Request and response headers can be customized. Lagom messages are serialised, by default, to ordinary JSON, using idiomatic mapping libraries that make it transparent how the JSON will be represented on the wire.
 
 ## Design driven APIs
 
-Lagom's interface specifications are designed to be compatible with design driven API approaches to system development.
-
-### Future plans
+ Lagom uses the same interfaces in Lagom to Lagom service communication. 
```

Comment:
I'm assuming this paragraph is WIP. Otherwise I think it needs a bit more explaining and maybe code samples or links to the ref guide to illustrate the statement.